PrebidMobile/PrebidMobileRendering/Utilities/UserAgentService.swift

```diff
@@ -38,18 +40,20 @@ public class UserAgentService: NSObject {
         }
         
         DispatchQueue.main.async {
-            self.webView.evaluateJavaScript("navigator.userAgent") { [weak self] result, error in
-                guard let self = self else { return }
-                
-                if let error {
-                    Log.error(error.localizedDescription)
-                }
-                
-                if let result = result, self.userAgent.isEmpty  {
-                    self.userAgent = "\(result)"
+            if let webView = self.webView {
```

Comment:
If the webview has not initialized the completion block will not be called. Is that OK?